doc/content/examples/example14.md

```diff
@@ -0,0 +1,258 @@
+# Example 14: Seismic analysis of a base-isolated nuclear power plant building
+
+!alert note title=Units of this example
+GN, GPa, m, and sec
+
+This example demonstrates the seismic analysis of a nuclear power plant (NPP) building that houses a reactor vessel and four steam generators. This NPP building is identical to that of [Example 13](LINK) and is a hypothetical molten-chloride fast reactor (NPP). Details of the NPP building are provided in [Example 13](LINK) as well as [!citet](inl-ext-20-59608). This example is presented in two parts: (1) seismic analysis of just the foundation basemat on seismic isolators, and (2) seismic analysis of the full building on seismic isolators. The foundation basemat and the isolation system in both cases is identical. In each case, the modeling is briefly described and the results are presented. The mesh for both the parts of this example is generated in Cubit and the Cubit journal file (`fixed_base_with_isolators_new.jou`) for these meshes is also included in the folder of this example in the repository. 
+
+## Part 1: Foundation basemat analysis
+
+### Modeling
+
+Before performing the seismic analysis of the full NPP building, the base-isolated basemat is first analyzed. The isolation system of the building comprises 70 Friction Pendulum^TM^ (FP) isolators arranged in a grid shown in [fig:iso_plan] below. Note that the figure shows a view of the foundation from the bottom, and therefore the +Z axis is pointing inwards into the basemat. The basemat is 98.75m x 68.75m in plan and is 1m thick.
+
+!row!
+
+!col! small=12 medium=6 large=6
+!media media/examples/ex14/isolation_plan.png
+       style=width:100%;margin-right:0px;float:center;
+       id=fig:iso_plan
+       caption=Isolators under the basemat of the NPP building analyzed in Part 1 of this example
+!col-end!
+
```

Comment:
I am not sure why, but I am seeing that the figure numbers on the site are not appearing in order.

Reply:
yeah I noticed that too. I don't know why. I think its ok for now. They are linked correctly though.